[2024-09-11T01:51:23.991+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-09-11T01:51:24.059+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_first_dag_v4.first_task scheduled__2022-03-31T00:00:00+00:00 [queued]>
[2024-09-11T01:51:24.073+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_first_dag_v4.first_task scheduled__2022-03-31T00:00:00+00:00 [queued]>
[2024-09-11T01:51:24.076+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 6
[2024-09-11T01:51:24.176+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): first_task> on 2022-03-31 00:00:00+00:00
[2024-09-11T01:51:24.196+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v4', 'first_task', 'scheduled__2022-03-31T00:00:00+00:00', '--job-id', '723', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/tmp/tmpjevn6zcq']
[2024-09-11T01:51:24.191+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=3775) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-09-11T01:51:24.200+0000] {standard_task_runner.py:63} INFO - Started process 3791 to run task
[2024-09-11T01:51:24.199+0000] {standard_task_runner.py:91} INFO - Job 723: Subtask first_task
[2024-09-11T01:51:24.307+0000] {task_command.py:426} INFO - Running <TaskInstance: our_first_dag_v4.first_task scheduled__2022-03-31T00:00:00+00:00 [running]> on host 0359316e7bd1
[2024-09-11T01:51:24.577+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='vohoang' AIRFLOW_CTX_DAG_ID='our_first_dag_v4' AIRFLOW_CTX_TASK_ID='first_task' AIRFLOW_CTX_EXECUTION_DATE='2022-03-31T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2022-03-31T00:00:00+00:00'
[2024-09-11T01:51:24.578+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-09-11T01:51:24.607+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-09-11T01:51:24.609+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo "Hello my son. I\'m your dad."']
[2024-09-11T01:51:24.627+0000] {subprocess.py:86} INFO - Output:
[2024-09-11T01:51:24.630+0000] {subprocess.py:93} INFO - Hello my son. I'm your dad.
[2024-09-11T01:51:24.632+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-09-11T01:51:24.634+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-09-11T01:51:24.689+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v4, task_id=first_task, run_id=scheduled__2022-03-31T00:00:00+00:00, execution_date=20220331T000000, start_date=20240911T015124, end_date=20240911T015124
[2024-09-11T01:51:24.827+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-09-11T01:51:24.916+0000] {taskinstance.py:3503} INFO - 2 downstream tasks scheduled from follow-on schedule check
[2024-09-11T01:51:24.927+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
